loader/helpers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const DEFAULT_DIRECTORY = 'test';
+
+function getDirectory(rc) {
+
+  if (!(typeof rc === 'object' && rc.directory)) {
+    return DEFAULT_DIRECTORY;
+  }
+
+  const dir = rc.directory;
+  const last = dir.substring(dir.length - 1);
+
+  // strip trailing slashes
+  if (last === '/' || last === '\\') {
+    return dir.substring(0, dir.length - 1);
+  }
+
+  return dir;
+}
+
+function checkDirectory(dir) {
+
+  dir = path.resolve(dir);
+  try {
+    fs.accessSync(dir);
+  } catch (e) { // eslint-disable-next-line max-len
+    e.message = `(clutch-assert/loader) Tried to instrument ${dir} but it does not exist.
```

Comment:
I'm implicitly using `fs.F_OK`to only check if the directory exists, if the directory cannot be read be the process the test runner will throw an error before this code is executed. 

Resolution: Explicitly use the constant to make it clearer as to what the code is doing